app/api/v1/endpoints/auth.py

```
@@ -587,16 +587,19 @@ async def get_oauth_url(
 
 
 
-@router.get("/oauth/callback")
+@router.get("/oauth/callback/google")
 async def oauth_callback(
     code: str,
     state: str,
     client_request: Request,
     db: AsyncSession = Depends(get_database)
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)